### ISSUE TITLE
Execution of DSpot maven plugin dosn't work on master branch

### ISSUE THREAD

**Issue by nicolabertazzo** [STAMP-project/dspot#378]
#### Characteristics
- **Issue Type**: [bug]
- **Reproducibility**: [always]
- **Severity**: [major]
- **Tool/Service/Component**: [dspot-maven, 167fffc]
- **Reporter**: [Nicola Bertazzo, nicola.bertazzo@eng.it]

#### Description
I'm trying to execute dspot from maven plugin but I have some problem.
If I execute dspot from maven, using the follow command, using the current master version (fc0eb26) of dspot, I have the error that I attached to this issue (resourceAsStream is null).
[error.txt](https://github.com/STAMP-project/dspot/files/1851929/error.txt)

#### Steps to reproduce
Clone dhell:
```
git clone https://github.com/STAMP-project/dhell.git
```
Access to dhell project folder:
```
cd dhell
```
execute dspot from maven:
```
clean package  fr.inria.stamp:dspot-maven:1.0.7-SNAPSHOT:amplify-unit-tests -Dfilter=eu.stamp.examples.dhell*
```

#### [Help on issue template](https://github.com/STAMP-project/dspot/blob/master/.github/ISSUE_DOC.md) 
Preview to follow the link or open file .github/ISSUE_DOC.md

**Comment by danglotb** [comment]
Hi @nicolabertazzo 

Does the other test pass with your modification? 

Will propose your fix by opening a pull request?

Thank you very much! 

-- Benjamin.

**Comment by danglotb** [comment]
Closed with #380, Thank you @nicolabertazzo !
